### ISSUE TITLE
Cache translations / localizations

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#1030]
### Problem

Currently, each time the page loads it must request the fairly large [local json](https://github.com/okTurtles/group-income-simple/blob/master/strings/english.json) (at the moment on the order of 38kb).

### Solution

Explore ways of *not* loading this file.

There are two main approaches:

- **Using caching headers, the traditional way.** Check to see if the Hapi server is using proper headers to prevent the client from downloading the json each time. It might be that browsers can automatically cache this stuff for us without having to write any code. It might even already do that. I just haven't checked.
- **Use ServiceWorker caching.** This is an advanced technique that manually stores the json locally using the [`Cache`](https://developer.mozilla.org/en-US/docs/Web/API/Cache) API of ServiceWorkers. This requires manually storing it in the cache, keeping track of new versions of the json, etc.

To summarize: this might not be a real issue. Someone needs to verify what the current behavior is with the Hapi.js server and see if the traditional approach suffices for our needs.

We want the app to work well offline, and for this we might need to use the ServiceWorkers API. See related issue: #590

**Comment by taoeffect** [comment]
See also comment here: https://github.com/okTurtles/group-income-simple/pull/1098#pullrequestreview-758813456

Based on conversations on Slack with @snowteamer it sounds like changes to the translation `.json` files aren't reflected unless there's a restart of the backend, which is actually the behavior we want. I'm not entirely sure whether this issue can be closed now or not, but I'll leave it open just to remind us to double-check this behavior after release.

**Comment by taoeffect** [comment]
PR #1098 might've been sufficient to close this issue. We'll need to test it a bit more and verify.
